Vizzuality/landgriffon
# Landgrif 1404 intervention separate t1supplier and producer

### General description

This PR separates the 'suppliers' field in the intervention form in two different fields: 'T1 suppliers' and 'producers' 

### Testing instructions

1. Go to the intervention form. There should be a field for 'T1 suppliers' and other for 'producers'
2. The fields should have smart search
3. The fields should be dependent of and influence the other fields values 

### Task
https://vizzuality.atlassian.net/browse/LANDGRIF-1404

## Checklist before merging
